basyx/aas/adapter/http.py

```diff
@@ -669,6 +712,12 @@ def handle_request(self, request: Request):
             except werkzeug.exceptions.NotAcceptable as e:
                 return e
 
+    # ------ all not implemented ROUTES -------
+    def not_implemented(self, request: Request, url_args: Dict, **_kwargs) -> Response:
+        response_t = get_response_type(request)
+        raise NotImplemented(f"This route is not implemented!")
+        return response_t()
```

Comment:
This line `return response_t()` is unreachable, as we always raise `NotImplemented` before. Thus, it can be removed.